src/components/App.tsx

```diff
@@ -44,115 +32,160 @@ export default function App() {
+          </button>
+          {capitalizeFirstLetter(chain.name)}
         </div>
-        <div className="md:flex flex-row flex-wrap justify-center lg:pt-8">
-          <Card>
-            <div className="flex justify-between mx-8 py-8 border-b-2 border-gray-200 ">
-              <h1
-                  className="text-2xl lg:text-4xl"
-                  style={{color: extraInfo?.color}}
-              >
-                Metadata #{specs.metadataVersion}
-              </h1>
-              <div className="flex border-2 border-black rounded-xl p-2">
-                {specs.metadataQr.signedBy ? (
+      </div>
+      <div className="flex h-[91vh] md:h-auto">
+        <Sidebar
+          allChains={allChains}
+          sidebarStyle={sidebarStyle}
+          currentNetwork={currentNetwork}
+          setLocalNetwork={setLocalNetwork}
+          setCurrentNetwork={setCurrentNetwork}
+          color={color}
+        />
+        {/** darker layer*/}
+        {isOpen && (
+          <div
+            className="absolute w-full h-[91vh] bg-black opacity-80 z-20 visible"
+            onClick={() => {
+              setIsOpen(!isOpen);
+            }}
+          />
+        )}
+        {/** Main content */}
+        <div className="m-auto flex flex-col md:absolute md:left-60 md:pl-20 md:m-0">
+          <div className="md:flex flex-row flex-wrap justify-center md:pt-8">
+            <div
+              className="px-2 py-2 rounded-lg border-gray-600 bg-white text-black"
+              style={{ minWidth: "25rem" }}
+            >
+              <div className="flex justify-between mx-8 py-8 border-b-2 border-gray-200 items-center">
+                <h1 className="text-lg md:text-2xl" style={{ color }}>
+                  Metadata #{chain.metadataVersion}
+                </h1>
+                <div className="flex border-2 border-black rounded-xl p-2">
+                  {metadataQr.signedBy ? (
                     <div className="text-black font-normal">
-                      <BadgeCheckIcon className={svgClass}/>
-                      Signed by {specs.metadataQr.signedBy}
+                      <BadgeCheckIcon className={svgClass} />
+                      Signed by {metadataQr.signedBy}
                     </div>
-                ) : (
+                  ) : (
                     <div className="text-red-500">
-                      <ExclamationCircleIcon className={svgClass}/>
+                      <ExclamationCircleIcon className={svgClass} />
                       Unsigned
                     </div>
-                )}
+                  )}
+                </div>
               </div>
-            </div>
-            <div className="lg:flex grid justify-center pt-8">
-              <QrCode {...specs.metadataQr} />
-              <div className="text-black p-5 w-72">
-                <Specs chainSpecs={{...specs}} color={extraInfo?.color}/>
-                <AddToSigner {...specs.specsQr} />
-                <NextMetadata {...specs} />
```

Comment:
We need to have it. That's an important feature 

<img width="312" alt="image" src="https://user-images.githubusercontent.com/7165377/172148985-4b8d1576-0ad4-47b6-ad4a-c6ad8103025f.png">